src/data/navigation/sections/events.js

```diff
@@ -25,6 +25,30 @@ module.exports = [
       }
     ]
   },
+  {
+    title: "Journaling Tutorial",
+    path: "/events/journaling-tutorial/index.md",
+    pages: [
+      {
+        title: "How it works",
+        path: "/events/journaling-tutorial/index.md"
+      },
+   
+      {
+        title: "Register Commerce Events in Adobe Developer Console for Journaling",
+        path: "/events/journaling-tutorial/registration-jounrnaling-events.md"
```

Comment:
Typo: In the `registration-jounrnaling-events.md` filename `jounrnaling` needs to be replaced with `journaling`, and this path needs to be updated accordingly